### ISSUE TITLE
Login popup popups after login

### ISSUE THREAD

**Issue by subpub** [yaxim-org/yaxim#250]
on fresh install(Do this within a soon/minute) **>**  enter wrong password SEND **>** maybe try another time **>** third time correct one **>** OK! **>**  you'll get the popup again

**Comment by ge0rg** [comment]
Yeah, the first startup dialog is a little mess, full of race conditions. I really need to fix all that handling. Thanks for reminding, I'll keep this issue open... it might be a while.
